examples/dreamcast/basic/exec/Makefile

```diff
@@ -26,7 +26,6 @@ $(SUBTARGET): $(SUBOBJS)
 	kos-cc -o sub.elf $(SUBOBJS)
 	$(KOS_OBJCOPY) -O binary sub.elf $(SUBTARGET)
 
-# You can safely remove the next two targets if you don't use a ROMDISK
```

Comment:
Doesn't this one need `KOS_ROMDISK_DIR`? You probably still need to keep that romdisk.img has a dependency on the `$(SUBTARGET)`, of course.

Reply:
I didnt include KOS_ROMDISK_DIR to makefiles that have custom romdisk dependencies.  That includes this example (which seems broken btw because it doesnt load the ), bumpmap, and modifier volume with te. ture. I'll pull the latest changes and work on them.

Reply:
Ah... Well, in that case, they'd still need the KOS_INIT_ROMDISK in the code, if they are to keep their old special target stuff.

Reply:
I got it all sorted out and pushed my changes.  The three examples I mentioned look to be working.